package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bloom-housing/ui-components",
-  "version": "7.2.2",
+  "version": "8.2.0",
```

Comment:
Wouldn't have expected to see a version edit manually here

Reply:
oh, that was a left over from testing. Since the linking wasn't working, I was trying a tool called [yalc](https://github.com/wclr/yalc), but ran into the same issues. I had to make the version number be the version I wanted to fake publish. I'll revert this back to the old version as we are no longer updating this anywhere